interface-definitions/interfaces-wwan.xml.in

```diff
@@ -36,6 +36,67 @@
           <leafNode name="mtu">
             <defaultValue>1430</defaultValue>
           </leafNode>
+          <node name="ping-check">
+            <properties>
+              <help>Use ping check rather than interface state check for automatic re-dial</help>
+            </properties>
+            <children>
+              <leafNode name="enable">
```

Comment:
Rhis feels superfluous - if ping-check CLI node is present, that‘s evidence enough to enable it

Reply:
Should I remove this node or change it to disable?

My though process is that the user may want to keep the config but doesn't have it enabled at time.

Reply:
If you think a knob is a good idea please use disable instead

Reply:
Personally don't think such a small node requires a disable/enable value.

It makes sense when you have many values to add/remove in order to toggle.